assets/css/base/_base.scss

```
@@ -1,7 +1,10 @@
+@import 'typography';
+
 body {
   border-top: 1rem solid $primary-color;
   font-family: $sans-serif;
-  font-size: pxToRem(20);
+  font-size: var(--step0);
+  line-height: $line-height-heading;
```

Why are you adding `$line-height-heading` to the body element? I see that you are overwriting this in multiple places down below. I would propose adding `$line-height-body` here (because it’s the body) and delete all places were you are setting `$line-height-body` on other elements. Headlines etc. can still have the deviating line-height. This way we let the cascade of CSS work for us.

```suggestion
  line-height: $line-height-body;
```